src/app/entry/delete/dialog/delete-entry-dialog.component.html

```diff
@@ -0,0 +1,24 @@
+<h1 mat-dialog-title>Are you sure you want to delete this Dockstore {{ term }}?</h1>
+<div mat-dialog-content>
+  <app-alert></app-alert>
+  <p>
+    You are about to permanently delete the {{ term }} <b>{{ path }}</b
+    >. After you delete the {{ term }}, it will no longer appear on Dockstore.
+  </p>
+  <p>A {{ term }} can be deleted if it:</p>
+  <ul>
+    <li>Has never been published,</li>
```

Comment:
Do these 3 restrictions apply to non-GitHub app entries? I don't know. But if they don't apply, then I think you'd need the `*ngIf`.

Reply:
Yup, the restrictions apply to all entries.